app/src/features/surveys/view/SurveyAnimals.tsx

```diff
@@ -182,16 +181,16 @@ const SurveyAnimals: React.FC = () => {
         <EditDialog
           dialogTitle={
             <Box>
-              <HelpButtonTooltip content={SurveyAnimalsI18N.animalIndividualsHelp}>
-                <Typography variant="h3">Individuals</Typography>
+              <HelpButtonTooltip
+                content={SurveyAnimalsI18N.animalIndividualsHelp}
+                iconSx={{ position: 'relative', top: '-4px', right: '0px' }}>
+                <Typography variant="h3" component="span">{`${animalFormMode} Animal`}</Typography>
```

Comment:
I wouldn't directly render the value of an enum, since a) it's often not in a suitable format to print, and b) it's subject to change.

```javascript
<Typography variant="h3" component="span">
  {animalFormMode === ANIMAL_FORM_MODE.EDIT ? 'Edit Animal' : 'Add Animal'}
</Typography>
```

Reply:
Those are some good points, I'll revert